src/main/kotlin/domain/cars/Cars.kt

```diff
@@ -0,0 +1,26 @@
+package domain.cars
+
+import Constants
+import domain.numbergenerator.NumberGenerator
+
+class Cars(input: String) {
+    private val cars: List<Car> = CarsArgumentConverter.toCarList(input)
+
+    fun lap(numberGenerator: NumberGenerator) {
+        cars.forEach { it.moveOrStop(numberGenerator) }
+    }
+
+    fun getWinners(): String {
+        val maxDistance = cars.maxOf { it.distance }
+        val winnerNames = cars.filter { it.distance == maxDistance }.map { it.carName }
+        return winnerNames.joinToString(Constants.WINNER_NAME_DELIMITER) { it }
+    }
```

Comment:
`{ it }`를 쓰는 것과, 생략하는 것과는 무엇이 다른가요?
```suggestion
        return winnerNames.joinToString(Constants.WINNER_NAME_DELIMITER)
    }
```

Reply:
코틀린에서 it 라는 키워드로 원소를 순회할 수 있도록 제공하는 데 현재 상황에서는 생략하는 것이 좋을 것 같습니다.
원소를 명시하고 싶다면 it 보다는 의미있는 이름으로 `{ winnerName -> winnerName }` 과 같이 사용할 것 같습니다.